Add index for 1.0 migration guide (#820)
Qiskit/documentation: docs/api/migration-guides/index.mdx

> We've prepared various migration guides to help you more effectively use Qiskit&reg; and Qiskit Runtime:

Reviewer: ```suggestion
These migration guides help you more effectively use Qiskit&reg; and Qiskit Runtime:
```

Reviewer: (removing "various" to make it less vague, and rewording to avoid the "we")

* Migrate to Qiskit 1.0

>    * [Qiskit 1.0 packaging changes](/api/docs/migration-guides/qiskit-1.0-installation)

Reviewer: ```suggestion
   * [Qiskit 1.0 packaging changes](/api/migration-guides/qiskit-1.0-installation)
```

>    * [Qiskit 1.0 feature changes](/api/docs/migration-guides/qiskit-1.0-installation)

Author: Ooops thank you
